jobs/import-to-patient-last-update-date.yml

```diff
@@ -0,0 +1,31 @@
+# Template that takes in the following parameters
```

Comment:
This job loads the dbpatient_event table into the warehouse as a table named "patient_last_update_date".  It is partitioned, so can be used to query the most recent date a particular patient on a given server was last updated.